lib/ui/views/installer/installer_viewmodel.dart

```diff
@@ -212,10 +245,9 @@ class InstallerViewModel extends BaseViewModel {
       'Integration source: ${_managerAPI.getIntegrationsRepo()}',
       
       '\n- Logs',
-      logs,
```

Comment:
Why not put logsTrimmed here instead of appending it in the clipboard copy?

Reply:
Because the I would have to write .join('\n').

Reply:
Although it might be cleaner?

Reply:
Maybe I should do that....

Reply:
There, changed it.

Reply:
Are you sure you need to join \n here? it's still being joined by the clipboard copy anyways

Reply:
Nvm